gradle/dependencies.gradle

```diff
@@ -21,7 +21,7 @@ def versions = [
     dokka: '0.9.17',
     errorProne: '2.3.1',
     errorPronePlugin: '0.0.13',
-    kotlin: '1.2.21',
+    kotlin: '1.2.51',
```

Comment:
Want to try 1.3-M1 in dev branch?

Reply:
No - we distribute kotlin artifacts, so we can't target unstable versions

Reply:
We definitely shouldn't ship it, but it can be a good idea if unstable/snapshot will target upcoming version, so we can eagerly target compatibility.

Reply:
In general we like to keep master stable for contributors, so unlikely to check in any unstable versions :/